contracts/.env.example

```diff
@@ -0,0 +1,12 @@
+# Ethereum config
+PRIVATE_KEY=
+ETHEREUM_RPC_URL=
+ETHERSCAN_API_KEY=
+
+# TendermintX config
+HEADER_RANGE_FUNCTION_ID=
+ROTATE_FUNCTION_ID=
+GENESIS_HEIGHT=
+GENESIS_HEADER=
+GENESIS_AUTHORITY_SET_ID=
```

Comment:
how do they get these quantities? maybe leave a comment saying that they can fill in these later?

Reply:
Added script for getting the genesis parameters.